modules/twinkleblock.js

```diff
@@ -358,8 +426,35 @@ Twinkle.block.callback.change_action = function twinkleblockCallbackChangeAction
 	if (field_block_options) {
 		oldfield = $form.find('fieldset[name="field_block_options"]')[0];
 		oldfield.parentNode.replaceChild(field_block_options.render(), oldfield);
+
+
+		$form.find('[name=pagerestrictions]').select2({
+			width: '100%',
+			tags: true,
+			placeholder: 'Enter pages to block user from',
+			maximumSelectionLength: 10, // Software limitation [[phab:T202776]]
+			language: { noResults: function() {
+				return 'No pages entered yet';
+			} }
+		});
```

Comment:
> Can't stop multiselect from showing dropdown: ... 

You're talking about this, right?

To hide the dropdown, simply add `templateResult: function() {}` . Then, the only thing that appears would be the noResults text, which can be hidden by adding to the CSS `.select2-results__message[role=alert] { display: none; }` .

Reply:
`select2-results__message[role=alert]` is perfect!  Maybe I just wasn't looking closely at the source, but I didn't see a way to hide it for pages but not namespace.  Very nice!

I still dislike how hiding the results slightly changes the layout of the box even with no results — rounded edge to angled — but I think that's unavoidable.

Reply:
The comment you just added might need updating, as `.select2-results__message[role=alert] { display: none; }` only hide the noResults text. The search hits are being hidden by `templateResult: function() {}`.

But not so easy, there's a problem here:
1. Enter "aaaaa" and press enter. It gets added.
2. Now enter "aa" (or any substring of any already selected choice) and press enter. Doesn't work :( 

Will have to tweak something here.

Reply:
Ahh, that's a good catch.  It's still behaving normally, but it defaults to existing options.  Since "a" shows up above "aaaa", if you press the up arrow, it will work, but that's obviously untenable.  It feels like this should be doable by tweaking the matcher or something, but it may be best to just revert to how I had it before and let well enough alone.